source/bucket.go

```diff
@@ -1,30 +0,0 @@
-package source
-
-import (
-	"fmt"
-
-	platform "github.com/influxdata/influxdb/v2"
-	"github.com/influxdata/influxdb/v2/http"
-	"github.com/influxdata/influxdb/v2/http/influxdb"
-	"github.com/influxdata/influxdb/v2/tenant"
-)
-
-// NewBucketService creates a bucket service from a source.
-func NewBucketService(s *platform.Source) (platform.BucketService, error) {
```

Comment:
This function was used in the launcher to build the `http/api_handler.go` `APIBackend` struct, but it was never used anywhere after that.